### PR TITLE
Make the scheduler aware of KEVs

### PR DESCRIPTION
**Pull request by felddy**
#  #

## 🗣 Description ##

This PR makes the scheduler aware of the KEV field in tickets.
As per the requirements, when a host has a ticket with `details.kev` set to `true` the scheduler will schedule this with the same priority as a critical vulnerability.

## 💭 Motivation and context ##

These changes makes no sense.  I have no idea why we are doing it.  See: cisagov/cyhy-system#38
All vulnerabilities that end up in the "Known Exploited" catalog will already have high priorities from their CVSS scores.

Closing language:
- Placates and closes cisagov/cyhy-system#38
- Closes cisagov/cyhy-system#44
- Closes cisagov/cyhy-system#45  

## 🧪 Testing ##

- [x] Tested by scanning a host with a vulnerability that is listed in the KEV but no other concurrent criticals on the same host.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Excellent 👍

File: cyhy/db/scheduler.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from datetime import datetime
 
 from cyhy.core.common import *
-from cyhy.db.queries import max_severity_for_host
+from cyhy.db.queries import max_severity_for_host, kev_count_for_host
```
Comment (mcdonnnj) [inline cyhy/db/scheduler.py]:
```suggestion
from cyhy.db.queries import kev_count_for_host, max_severity_for_host
```

File: cyhy/db/queries.py [hunk 2]
```diff
@@ -269,6 +269,18 @@ def max_severity_for_host(ip_int):
     )
 
 
+def kev_count_for_host(ip_int):
+    """Return the number of open tickets that have kev set to true in its details."""
```
Comment (mcdonnnj) [inline cyhy/db/queries.py]:
```suggestion
    """Return the number of open tickets that have kev set to true in their details."""
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Straightforward 👍 I had some housekeeping suggestions for your consideration.

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**